integration/bootstrapping_test.go

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+//go:build integration
+
+package integration
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/caarlos0/env/v6"
+	"github.com/eclipse-kanto/kanto/integration/util"
+	"github.com/eclipse/ditto-clients-golang"
+	"github.com/eclipse/ditto-clients-golang/model"
+	"github.com/eclipse/ditto-clients-golang/protocol"
+	"github.com/eclipse/ditto-clients-golang/protocol/things"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	bootstrappingFeatureID             = "Bootstrapping"
+	actionResponse                     = "response"
+	msgFailedCreateWebsocketConnection = "failed to create websocket connection"
+	eventFilterTemplate                = "like(resource:path,'/features/%s/*')"
+	sbServiceName                      = "suite-bootstrapping.service"
+	suiteConnector                     = "suite-connector.service"
```

Comment:
align it with the constant above, eg. `scServiceName` or `suiteConnectorServiceName`